using-timescaledb/continuous-aggregates.md

```diff
@@ -259,12 +259,14 @@ SELECT min_time::timestamp FROM device_summary;
 ---
 
 ### Future Work [](future-work)
-The first version of continuous aggregations has been released in TimescaleDB
-v1.3, but we have a number of new capabilities and improvements already planned
-in the next releases. Please find some of these forthcoming capabilities
-described below. If you'd like to help implement them, test them, want to help
-us prioritize, or have other areas you think we should work on, please get in
-touch via our [Github][timescale-github] or [Slack][support-slack].
+The first version of continuous aggregations was released in TimescaleDB
+v1.3, and we have a number of new capabilities and improvements already planned
+in the next releases. We support the ability to create multiple continuous 
```

Comment:
> We support the ability to create multiple continuous aggregates on a hypertable in TimescaleDB v1.4.

Shouldn't this line come before 

> and we have a number of new capabilities and improvements already planned in the next releases.

?

"Support for creating multiple aggregates on a hypertable was released in v1.4."?

Reply:
modified.